### PR TITLE
WIP Add representer repo status to track dashboard

### PR DESCRIPTION
**Pull request by Roshanjossey**
Address #31

 - Add new AsideItem for representer
 - Handle fetching, caching data for track representer

 Todo: Refactor TrackAside to make TrackAside component and icon generic

### CODE REVIEW

File: src/components/TrackAside.tsx [hunk 1]
```diff
@@ -220,3 +232,32 @@ function TestRunnerIcon({
...
+      onToggle={onToggleDetails}
+    >
+      <p>
+        This check passes if there is a <code>Dockerfile</code> file present in
+        the <code>exercism/{trackId}-representer</code> repository.
+      </p>
+
+      <p className="mb-0">
+        You can find more information about the <code>Dockerfile</code> file{' '}
+        <a href="https://github.com/exercism/automated-tests/blob/master/docs/docker.md">
```
[the remark this answers is not in the document]
Comment (SleeplessByte) [reply src/components/TrackAside.tsx]: You can leave it perhaps. Idk if we have one for representers. cc @ccare @iHiD

[the remark this answers is not in the document]
Comment (iHiD) [reply src/components/TrackAside.tsx]: No, we don't yet.

[the remark this answers is not in the document]
Comment (Roshanjossey) [reply src/components/TrackAside.tsx]: Got it @SleeplessByte , @iHiD . I'll add a `Todo` to change it in a future point.

[the remark this answers is not in the document]
Comment (iHiD) [reply src/components/TrackAside.tsx]: There are now files [here](https://github.com/exercism/automated-analysis/tree/master/docs/representers)

**Comment by SleeplessByte** [comment]
Superseeded by https://github.com/exercism/tracks-maintenance-dashboard/commit/7f3ef56de42d1fd74661c0749e1cfe988addaf69

**Pull request closed without merging**